python/hsfs/helpers/quicktours/feature_store.md

```diff
@@ -0,0 +1,71 @@
+# Feature Store API Quick Tour
+
+The `FeatureStore` serves as the central repository for features data and metadata in the Hopsworks Platform.
+
+## Get or Create Feature Groups to write to the Feature Store
+
+You can retrieve or create `FeatureGroup` objects using the `get_or_create_feature_group()` method. `FeatureGroup`s objects encapsulate metadata about a table in the Feature Store, such as its schema, and are used to insert new data in the `FeatureStore`.
+
+```python
+my_feature_group = fs.get_or_create_feature_group(
+    name="my_feature_group",
+    version=1,
+    description="my first feature group",
+    primary_key=['user_id'],
+    event_time='visited_last'
+    online_enabled=True
+)
+
+my_feature_group.insert(dataframe_with_feature_data)
+```
+
+## Get or Create Feature Views to Read Feature Data
+
+You can retrieve or create `FeatureView` objects using the `get_or_create_feature_view()` method. `FeatureView`s objects are a list of selected features from one or more `FeatureGroup`s to be used as input to a model. You can use it to materialize `TrainingDataset`s or serve **Real-Time** feature data to provide up to date context to your model.
+
+```python
+my_feature_view = fs.get_or_create_feature_view(
+    name="my_feature_view",
+    version=1,
+    description="my first feature view",
+    query=my_feature_group.select(["feature1", "feature2"])
+)
+
+x_train, x_test, y_train, y_test = fv.train_test_split()
+# or
+real_time_feature_data = fv.get_feature_vector(entry={"user_id": 1})
+```
+
+## Get or Create Storage Connectors to access External Data
+
+You can retrieve or create `StorageConnector` objects using the `get_or_create_storage_connector()` method. `StorageConnector`s objects are used to access external data sources, such as S3, Snowflake, BigQuery. Each `StorageConnector` has its own properties so check out the docs for the specific data source you are interested in:
```

Comment:
we dont have create storage connector method, there is only `get_storage_connector`